app/src/tests/components/contents/downloads/orderly/Artefacts.test.tsx

```diff
@@ -32,7 +32,7 @@ const renderComponent = (packet?: PacketMetadata) => {
           <Route element={<Outlet context={{ packet }} />}>
             <Route
               path="/:packetName/:packetId/downloads"
-              element={<Artefacts artefacts={mockPacket.custom?.orderly.artefacts as Artefact[]} />}
+              element={<Artefacts artefacts={packet?.custom?.orderly.artefacts as Artefact[]} />}
```

Comment:
I assume the same correction applies a few lines back on 30, too.

Reply:
Yeah, although slightly complicated by the fact that `packet` might be undefined so you still need to give the router something to route with in that case. I'm also going to change the "when packet is not found.." test to check absence of generic /Download/ text rather than a specific artefact name.